src/universe/me.ts

```diff
@@ -97,6 +102,23 @@ export class UniverseMe {
     }
   }
 
+  public async federatePrismaticToken (): Promise<PrismaticCredentials> {
```

Comment:
Nitpick: We should add JSDoc here to explain the endpoint so that SDK users can understand why it exists and what it gives hem.

Reply:
AFSIK product want to obfuscate that we are using iPaaS and what iPaaS we are using, so I'd align with product (@jakobwolffmartinez and @walmir) if we want to expose this in the general browser sdk.

Reply:
Prismatic. They support [auth code (RFC 6749, section 4.1)](https://tools.ietf.org/html/rfc6749#section-4.1) and [client credentials (RFC 6749, section 4.4)](https://tools.ietf.org/html/rfc6749#section-4.4), as well as Azure AD for SSO.

Reply:
I don't think we have the time to properly sync on it so I just follow your solution and knowledge now then. Sorry if it was a strange thing to bring into the conversation.